### ISSUE TITLE
Unable to access boards

### ISSUE THREAD

**Issue by apowell656** [foxythemes/jira-cli#11]
Looking at the documentation on Boards - https://docs.jiracli.com/#general_boards there is nothing clear on how to access a particular board. As it stands the jira command on lists all issues without the ability to filter based on status and I think that being able to view by boards would be useful.

**Comment by miguelmich** [comment]
@apowell656 you're right, I'll find some time next month to add the pending new features including this one :)

**Comment by apowell656** [comment]
That sounds great and I look forward to it.
